RetailCoder.VBE/UI/RubberduckUI.de.resx

```diff
@@ -1210,4 +1210,103 @@ Warnung: Alle eigenen Einstellungen gehen dabei verloren. Die Originaldatei wird
   <data name="SourceControl_DeleteBranchButton" xml:space="preserve">
     <value>Zweig löschen</value>
   </data>
+  <data name="ExtractInterface_Caption" xml:space="preserve">
+    <value>Rubberduck - Extract Interface</value>
+  </data>
+  <data name="DeselectAll_Button" xml:space="preserve">
+    <value>Alles abwählen</value>
+  </data>
+  <data name="SelectAll_Button" xml:space="preserve">
+    <value>Alles auswählen</value>
+  </data>
+  <data name="ExtractInterface_TitleLabel" xml:space="preserve">
+    <value>Extrahiere Interface</value>
+  </data>
+  <data name="ExtractInterface_MembersGroupBox" xml:space="preserve">
+    <value>Mitglieder</value>
+  </data>
+  <data name="ExtractInterface_InstructionLabel" xml:space="preserve">
+    <value>Bitte spezifizieren Sie den Interface-Namen und die Mitglieder </value>
+  </data>
+  <data name="IndentCurrentModule" xml:space="preserve">
+    <value>Aktuelles Projekt</value>
+  </data>
+  <data name="IndentCurrentProcedure" xml:space="preserve">
+    <value>Aktuelle Prozedur</value>
+  </data>
+  <data name="IndentCurrentProject" xml:space="preserve">
+    <value>Ganzes Projekt</value>
+  </data>
+  <data name="SmartIndenterMenu" xml:space="preserve">
+    <value>Einrücken</value>
+  </data>
+  <data name="RefactorMenu_IntroduceField" xml:space="preserve">
+    <value>&amp;Feld einfügen</value>
+  </data>
+  <data name="PromoteVariable_InvalidSelection" xml:space="preserve">
+    <value>Die Auswahl ist keine Variable</value>
+  </data>
+  <data name="RefactorMenu_EncapsulateField" xml:space="preserve">
+    <value>G&amp;ekapseltes Feld</value>
```

Comment:
Hier geht es um den Vorgang des Kapselns. "Feld kapseln" wäre ggf. passender. 
Gleiches gilt für die Überschrift
---
It's about the action of encapsulation here. "Feld kapseln" might be a better choice. 
Same applies for the caption.